### ISSUE TITLE
intel

### ISSUE THREAD

**Issue by DARK-DEVIL-66** [Bendr0id/xmrigCC#405]
this miner will work in intel cpu like i3/i5/i7 etc?

**Comment by Bendr0id** [comment]
Yes it will work, CPU is Auto detected and configured. Just start the miner and configure pool, wallet address in the config.json

**Comment by Bendr0id** [comment]
See this for sample:

https://github.com/Bendr0id/xmrigCC/wiki/Config-XMRigDaemon

**Comment by DARK-DEVIL-66** [comment]
can i use controll penal in vps ?
(ubuntu installed )

doe's it need port forwading ?

how many executable need to run in my mining pc ?

**Comment by Bendr0id** [comment]
Yes you can run the server of xmrigcc on a vps. On AWS/cloud you need the open the port on the firewall. On others the port is opened automatically.

On the mining PC you need to have xmrigMiner and xmrigDaemon but only need to start xmrigDaemon it will launch xmrigMiner for you
